README.md

```diff
@@ -15,6 +15,37 @@ Current status:
 
 See the [docs](https://github.com/MastodonC/witan.models.demography/blob/master/doc/intro.md) for more info about the methodology.
 
+## Development tools
+
+When combining functions into a model there are useful tools to take advantage of, thanks to dependencies for `witan.workspace-executor` and `witan.workspace-api`.
+
+* To visualise a model workflow, use the `view-workflow` function:
+
+You need to have `Graphviz` installed:
+
+-> Ubuntu: `> sudo apt-get install graphviz`
+
+-> MacOS: `> brew install graphviz`
+
+For any OS you should also be able to install it with Pip: `pip install graphviz`.
+
+Use it to visualise a [cohort-component-model](https://github.com/MastodonC/witan.models.demography/blob/master/src/witan/models/dem/ccm/models.clj) as follows:
```

Comment:
The order of this section is still a bit strange. And 'it' needs to be more precise.

Can I suggest:

```To visualise a model workflow, you need to have Graphviz installed: 

-> Ubuntu: > sudo apt-get install graphviz

-> MacOS: > brew install graphviz

For any OS you should also be able to install it with Pip: > pip install graphviz.

To use Graphiz to visualise a [cohort-component-model](https://github.com/MastodonC/witan.models.demography/blob/master/src/witan/models/dem/ccm/models.clj) call the view-workflow function as follows:```